Dynatrace/dynatrace-operator
# Renames platform value "google" to "google-marketplace"

# Description

Renames platform "google" to "google-marketplace" to be consistent with "google-autopilot" platform value.

## How can this be tested?

* Create a standard GKE cluster as well as an autopilot cluster* 
* Build and push the deployer image with `make images/gcr/deployer`
* Connect to the standard cluster
* Apply the application crd: `kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/application/master/deploy/kube-app-manager-aio.yaml`
* Deploy using the deployer image with `make deploy/gke/deployer`
* Operator should be deployed correctly
* Connect to the autopilot cluster
* Apply the application crd: `kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/application/master/deploy/kube-app-manager-aio.yaml`
* Deploy using the deployer image with `make deploy/autopilot/deployer`
* Operator should be deployed correctly

## Checklist
